configs/prod/sysadmin.service

```diff
@@ -1,6 +1,6 @@
 [Unit]
 Description=Sysadmin
-After=network.target syslog.target
+After=network-pre.target syslog.target
```

Comment:
why not `starry-sysadmin.target`
if it works for MT/relay - why not do exactly the same?
if there is a reason - document it in the actual commit